app/src/test/java/org/mozilla/fenix/settings/SettingsFragmentTest.kt

```diff
@@ -71,6 +68,9 @@ class SettingsFragmentTest {
 
         advanceUntilIdle()
 
+        mockkObject(FeatureFlags)
```

Comment:
As this is something global that could affect other tests, should we call `unmockkObject`  after the test finish ?

Reply:
Good catch! Thank you. I will do the cleanup for the test class.